examples-gallery/plot_ball_rolling_on_spinning_disc.py

```diff
@@ -372,11 +370,12 @@ def obj_grad(free):
 
 # %%
 # Find the optimal solution.
-
+initial_guess = np.load('ball_rolling_on_spinning_disc_solution.npy')
 solution, info = prob.solve(initial_guess)
 print('message from optimizer:', info['status_msg'])
 print('Iterations needed', len(prob.obj_value))
 print(f'Optimal h = {solution[-1]:.3e} sec')
+np.save('ball_rolling_on_spinning_disc_solution.npy', solution)
```

Comment:
This will now show up in the docs directory every time you build locally and will have to be manually removed. Either remove this line or include it in a comment.

Reply:
> This will now show up in the docs directory every time you build locally and will have to be manually removed. Either remove this line or include it in a comment.

I simply forgot to remove it. I wil include in a comment, so people can see how to save a good initial guess for later use.